spec/rspec/rails/example/system_example_group_spec.rb

```diff
@@ -58,6 +58,31 @@ module RSpec::Rails
           expect(example).to have_received(:driven_by).once
         end
       end
+
+      describe '#after' do
+        it 'sets the :extra_failure_lines metadata to an array of STDOUT lines' do
+          group = RSpec::Core::ExampleGroup.describe do
+            include SystemExampleGroup
+
+            before do
+              driven_by(:selenium)
+            end
+
+            def set_exception(*) # rubocop:disable Naming/AccessorMethodName
+            end
```

Comment:
I don't need this locally, what do you see without it? Can you point to a failing travis build? (You can find your old builds despite the rebasing FYI)

Reply:
Oh hey, look at that, I must have had something else in a weird state causing this to bomb out. Removed!